examples/kubernetes.tf

```diff
@@ -0,0 +1,15 @@
+resource  "komodor_kubernetes" "k8s_cluster" {
+    cluster_name = "cluster-test-diff"
+}
+
+data "komodor_kubernetes" "data_k8s_cluster" {
+  cluster_name = komodor_kubernetes.k8s_cluster.cluster_name
+}
+
+output "cluster_api_key" {
+  value = komodor_kubernetes.k8s_cluster.id
+}
+
+output "data_cluster_api_key" {
+  value = data.komodor_kubernetes.data_k8s_cluster.id
+}
```

Comment:
Why do we need those two outputs in the example? i think one of them is enough,
If we want to show that it is possible to get the ID from the resource or fetch it from komodor, then let's add a comment above each output to explain that only one should be used.

Reply:
Removed one of the outputs and added a comment.